packages/styles/scss/components/card/index.scss

```diff
@@ -638,7 +638,8 @@
     outline: none;
 
     .#{$prefix}--card__pictogram {
-      display: flex;
+      text-decoration: none;
+      color: $black-100;
```

Comment:
1. I fixed the 2 erroneous focus stages. Now we only have 1 focus that does the two actions required: it triggers the hover state and open the link on keyboard "enter/return" click
2. I have fixed the color property in the css file to now be **color: $black-100;** . I have added that piece of code because by wrapping the card around an anchor element made the text to be in blue and underlined